machine_management/control_plane_machine_management/cpmso-openstack-ts-az-config.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
```

Comment:
I don't understand this file. Where does the content come from? What's the relation with CPMS? And some of it is actually wrong (the first block mostly).

Reply:
This content was to address this particular comment: "[Troubleshooting the control plane machine set](https://docs.openshift.com/container-platform/4.13/machine_management/control_plane_machine_management/cpmso-troubleshooting.html): an OpenStack section should definitely be created here mentioning https://access.redhat.com/solutions/7013893 and https://access.redhat.com/solutions/7024383"

Reply:
@EmilienM @maxwelldb Can you let me know about the parts that needs to be removed or corrected here? Especially, the first block.

Reply:
@EmilienM any comments on the first block that you mentioned?

Reply:
Answered here: https://github.com/openshift/openshift-docs/pull/64454/files#r1357054628